pkg/cgroups/cgroups.go

```diff
@@ -45,13 +45,19 @@ func Find() (Subsystems, error) {
 	scanner := bufio.NewScanner(mountInfo)
 	for scanner.Scan() {
 		lineSegs := strings.Fields(scanner.Text())
-		if lineSegs[len(lineSegs)-3] != "cgroup" {
+		if len(lineSegs) != 11 {
```

Comment:
actually I'm pretty sure this change is incorrect. The reason for indexing from the end is that one of the fields is optional and can be missing altogether.